### DISCUSSION TITLE
How to cancel my pro-account

### DISCUSSION THREAD

**Discussion by MLDavies** [community/community#30159] [Other Feature Feedback, Questions, & Ideas]
I having been paying for pro-account. However, my settings say that I'm on a free account. I want to cancel the pro-account and stop paying for it, but I can't figure out how. Does anyone know how to do this?

**Comment by radar-blip** [comment]
Account or Billing Issues

[Support Page](https://support.github.com/request)

**Comment by Zoltus** [comment]
Hi! I hope u get this solved:) Here instructions I found in the wiki documents!

1. In the upper-right corner of any page, click your profile photo, then click Settings.
![image](https://user-images.githubusercontent.com/23402980/185734857-149da4f8-3e4b-4fc8-919d-76efe0781e3b.png)
2. In the "Access" section of the sidebar, click  Billing and plans.
3. Under "Current plan", use the Edit drop-down and click Downgrade to Free.
![image](https://user-images.githubusercontent.com/23402980/185734884-d7772029-0fb0-4d37-8617-ac24bb09a6f3.png)
